docs/docsite/rst/roadmap/COLLECTIONS_5.rst

```diff
@@ -0,0 +1,56 @@
+.. _ansible_4_roadmap:
+
+===================
+Ansible project 5.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_12` for the most recent updates on ``ansible-core``.
```

Comment:
The [draft](https://hackmd.io/y7BBcweNR3aRVLuMbKkDxw) included dates from ansible-core but they seem to have been removed from this proposal. We should adjust the phrasing or include the dates back here for easy reference.

Reply:
I copied this file from the Ansible 4 roadmap, which didn't have any of the core dates in it. I'm happy to add those if we want to maintain them going forward, but they do appear on the 2.12 roadmap, which is linked here in the first paragraph.

Reply:
Addressed in c740157718ec38db1f5cc2606478e72335dcad63.

Reply:
Well, if the core dates slip, the community package dates will slip too, right? So we'd need to update this page anyway.

Reply:
Yes, that's very likely.